Dockerfile

```diff
@@ -1,16 +1,46 @@
-FROM 		phusion/baseimage
-MAINTAINER 	Casey Bisson <casey.bisson@gmail.com
+FROM alpine:3.2
+MAINTAINER 	Casey Bisson <casey.bisson@gmail.com>
 
-# Update Apt
-RUN apt-get update
+# Alpine packages
+RUN apk --update \
+    add \
+        jq \
+        curl \
+        bash \
+        ca-certificates && \
+    curl -Ls https://circle-artifacts.com/gh/andyshinn/alpine-pkg-glibc/6/artifacts/0/home/ubuntu/alpine-pkg-glibc/packages/x86_64/glibc-2.21-r2.apk > /tmp/glibc-2.21-r2.apk && \
```

Comment:
> Putting aside @tgross 's hate for go get.

:laughing: 

> This would give you a version of Consul that was compiled for the platform. 

If we're going to do this, we should split the build container out from the container we ship though. A go build container is huge.